### PR TITLE
Remove polyfills, take I

### PR DESCRIPTION
**Pull request by lol768**
>    If cross-fetch includes a polyfill, that'd be an argument for going back to isomorphic-fetch :( It should be the implementing application that has the responsibility for including polyfills to avoid ending up with 4 different Promise implementations in the bundle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Server Pipe [![Build Status](https://travis-ci.com/UniversityofWarwick/js-serverpipe.svg?branch=master)](https://travis-ci.com/UniversityofWarwick/js-serverpipe)
 
-This is a basic wrapper for `cross-fetch` that we use to make authenticated requests back to the application server. This replaces
+This is a basic wrapper for `fetch()` that we use to make authenticated requests back to the application server. This replaces
```
Comment (matmannion) [inline README.md]: I don't think the second sentence makes sense any more here, it was added when we switched from isomorphic-fetch to cross-fetch but native fetch doesn't "replace" anything. I'd just drop the sentence entirely

File: README.md [hunk 2]
```diff
@@ -13,4 +13,6 @@ import 'core-js/modules/web.dom-collections.for-each';
 
 You will need:
 
-`npm i core-js@3.1.3`
\ No newline at end of file
+`npm i core-js@3.1.3`
+
+You will also need an appropriate `Promise` and `fetch()` polyfill.
```
Comment (matmannion) [inline README.md]: As we're recommending core-js can we just add the relevant polyfills to the code block above?

**Review by matmannion — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (lol768) [reply README.md]: We can suggest https://github.com/UniversityofWarwick/scholarships/blob/3f30d418219130b5c8f797423988780b31c491b3/app/assets/js/polyfills.js#L3 et al (DOM collection etc) but we'll still need a `fetch` polyfill

Perhaps we recommend https://github.com/github/fetch directly?

File: README.md (hunk 1 above)
Comment (lol768) [reply README.md]: This is done

File: README.md (hunk 2 above)
Comment (matmannion) [reply README.md]: Fine with me

**Pull request merged**